SignalMessaging/attachments/MessageApprovalViewController.swift

```diff
@@ -48,26 +54,13 @@ public class MessageApprovalViewController: OWSViewController, UITextViewDelegat
         super.viewDidLoad()
         self.navigationItem.title = NSLocalizedString("MESSAGE_APPROVAL_DIALOG_TITLE",
                                                       comment: "Title for the 'message approval' dialog.")
-    }
 
-    private func updateToolbar() {
-        var items = [UIBarButtonItem]()
-
-        let cancelButton = UIBarButtonItem(barButtonSystemItem: .stop, target: self, action: #selector(cancelPressed))
-        items.append(cancelButton)
-
-        if textView.text.count > 0 {
-            let spacer = UIBarButtonItem(barButtonSystemItem: .flexibleSpace, target: nil, action: nil)
-            items.append(spacer)
-            let sendButton = UIBarButtonItem(title: NSLocalizedString("SEND_BUTTON_TITLE",
-                                                                      comment: "Label for the send button in the conversation view."),
-                                             style: .plain,
-                                             target: self,
-                                             action: #selector(sendPressed))
-            items.append(sendButton)
-        }
+        self.navigationItem.leftBarButtonItem = UIBarButtonItem(barButtonSystemItem: .stop, target: self, action: #selector(cancelPressed))
+        self.navigationItem.rightBarButtonItem = sendButton
```

Comment:
> Similarly to the AttachmentApprovalVC, the MessageApprovalVC's custom toolbar-that-looks-like-a-navbar is now just a navbar.

Bear in mind that we're using three approval VCs now: attachment, message and contact.

Reply:
> One small change in behavior is that instead of hiding the "send" button when the text is empty, it remains visible, but we disable it. In general I think it's better to disable controls than to hide them.

Agreed!

Reply:
> Bear in mind that we're using three approval VCs now: attachment, message and contact.

Thankfully, the contact share approval was already using a standard navbar.